delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/SchemaImports.scala

```diff
@@ -0,0 +1,97 @@
+package ch.epfl.bluebrain.nexus.delta.sdk
+
+import cats.implicits._
+import ch.epfl.bluebrain.nexus.delta.rdf.IriOrBNode.Iri
+import ch.epfl.bluebrain.nexus.delta.rdf.Vocabulary.owl
+import ch.epfl.bluebrain.nexus.delta.rdf.jsonld.ExpandedJsonLd
+import ch.epfl.bluebrain.nexus.delta.sdk.SchemaImports.Fetch
+import ch.epfl.bluebrain.nexus.delta.sdk.model.ResourceRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.projects.ProjectRef
+import ch.epfl.bluebrain.nexus.delta.sdk.model.resolvers.ResolverResolutionRejection
+import ch.epfl.bluebrain.nexus.delta.sdk.model.resources.Resource
+import ch.epfl.bluebrain.nexus.delta.sdk.model.schemas.SchemaRejection.{InvalidSchemaResolution, WrappedResolverResolutionRejection}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.schemas.{Schema, SchemaRejection}
+import monix.bio.IO
+
+/**
+  * Resolves the OWL imports from a Schema
+  */
+final class SchemaImports private[sdk] (fetchSchema: Fetch[Schema], fetchResource: Fetch[Resource]) { self =>
+
+  /**
+    * Resolve the ''imports'' from the passed ''expanded'' document and recursively from the resolved documents.
+    *
+    * @param id         the schema id
+    * @param projectRef the project where the schema belongs to
+    * @param expanded   the schema expanded form
+    * @return a "fat-schema" with all the imports resolved
+    */
+  def resolve(id: Iri, projectRef: ProjectRef, expanded: ExpandedJsonLd): IO[SchemaRejection, ExpandedJsonLd] = {
```

Comment:
We also need the current caller here

Reply:
These details should be added once `Resolvers.fetchSchema` and `Resolvers.fetchResource` or an alternative is implemented.

It does not have much to do with `SchemaImport` lookup resolution. Let's add the necessary bits and pieces afterwards